public/js/office.web.js

```diff
@@ -128,7 +124,7 @@ $(() => {
 
   function isUserInVideoConference(){
     if($("#exampleModalCenter").data('bs.modal')){
```

Comment:
It can be cached to a variable to to avoid dom fetch like:

```
const modal = $("#exampleModalCenter")
if (modal.data('bs.modal') {
    modal...
}